### ISSUE TITLE
Pharo 10 errors

### ISSUE THREAD

**Issue by vitormcruz** [dionisiydk/Mocketry#33]
On the new 10th version of Pharo Mocketry fails while creating a new Mock stub or stub:

```
OrderedCollection(Object)>>errorSubscriptBounds:
OrderedCollection>>at:
SpNotebookPresenter>>selectPageIndex:
StObjectInspectorPresenter>>updatePages
StObjectInspectorPresenter>>refresh
StObjectInspectorPresenter>>initializePresenters
StObjectInspectorPresenter(SpPresenter)>>initializePrivateHooks
StObjectInspectorPresenter(SpPresenter)>>initialize
StObjectInspectorPresenter>>initialize
StObjectInspectorPresenter class(SpPresenter class)>>newApplication:owner:model:
StObjectInspectorPresenter class(StPresenter class)>>owner:on:
StInspector(SpPresenter)>>instantiate:on:
StInspector>>newInspectorFor:
[ :aModel | self newInspectorFor: aModel ] in StInspector>>initializePresenters in Block: [ :aModel | self newInspectorFor: aModel ]
SpMillerColumnPresenter>>newPresenterFor:
SpMillerColumnPresenter>>pushModel:
SpMillerColumnPresenter>>setRootModel:
StInspector>>initializePresenters
StInspector(SpPresenter)>>initializePrivateHooks
StInspector(SpPresenter)>>initialize
StInspector class(SpPresenter class)>>newApplication:model:
StInspector class(SpPresenter class)>>on:
StInspector class>>onObject:
StInspector class>>openOn:
SpCodeInspectItCommand>>inspectObject:
[ :result | 
		self inspectObject: result ] in SpCodeInspectItCommand>>execute in Block: [ :result | ...
SpCodeInspectItCommand(SpCodeSelectionCommand)>>evaluate:andDo:
SpCodeInspectItCommand(SpCodeSelectionCommand)>>evaluateSelectionAndDo:
SpCodeInspectItCommand>>execute
SpCommand(CmCommandDecorator)>>execute
```

```
GHMetaMessages class(ProtoObject)>>primitiveFailed:
GHMetaMessages class(ProtoObject)>>primitiveFailed
GHMetaMessages class>>setClass:to:
ByteString(ProtoObject)>>injectGHMutation:
ByteString class(GHObjectMutation)>>mutate:
ByteString(Object)>>stub
UndefinedObject>>DoIt
[ receiver withArgs: (context ifNil: [ #() ] ifNotNil: [ {context} ]) executeMethod: self compileDoit] in OpalCompiler>>evaluate in Block: [ receiver withArgs: (context ifNil: [ #() ] ifNot...etc...
FullBlockClosure(BlockClosure)>>on:do:
OpalCompiler>>evaluate
[
		oldBindings := self interactionModel bindings copy.
		receiver := self interactionModel doItReceiver.
		result := receiver class compiler
			source: aString readStream;
			context: self interactionModel doItContext;
			receiver: self interactionModel doItReceiver;
			requestor: self interactionModel;
			environment: self environment;
			failBlock:  [ ^ compileErrorBlock value ];
			evaluate.
		oldBindings size = self interactionModel bindings size 
			ifFalse: [ self withAdapterDo: [ :anAdapter | anAdapter refreshStyling ] ].
		result ] in SpCodePresenter>>evaluate:onCompileError:onError: in Block: [...
FullBlockClosure(BlockClosure)>>on:do:
SpCodePresenter>>evaluate:onCompileError:onError:
SpCodeInspectItCommand(SpCodeSelectionCommand)>>evaluate:andDo:
SpCodeInspectItCommand(SpCodeSelectionCommand)>>evaluateSelectionAndDo:
SpCodeInspectItCommand>>execute
SpCommand(CmCommandDecorator)>>execute
[ self decoratedCommand execute ] in SpToolCurrentApplicationCommand>>execute in Block: [ self decoratedCommand execute ]
[ activeProcess
			psValueAt: index
			put: anObject.
		aBlock value ] in SpToolCurrentApplication(DynamicVariable)>>value:during: in Block: [ activeProcess...
FullBlockClosure(BlockClosure)>>ensure:
SpToolCurrentApplication(DynamicVariable)>>value:during:
SpToolCurrentApplication class(DynamicVariable class)>>value:during:
SpToolCurrentApplicationCommand>>execute
[ self decoratedCommand execute ] in SpToolCurrentApplicationCommand>>execute in Block: [ self decoratedCommand execute ]
[ activeProcess
			psValueAt: index
			put: anObject.
		aBlock value ] in SpToolCurrentApplication(DynamicVariable)>>value:during: in Block: [ activeProcess...
FullBlockClosure(BlockClosure)>>ensure:
SpToolCurrentApplication(DynamicVariable)>>value:during:
SpToolCurrentApplication class(DynamicVariable class)>>value:during:
SpToolCurrentApplicationCommand>>execute
[ 
			aCmCommand canBeExecuted 
				ifTrue: [ aCmCommand execute ] ] in SpKMCategoryBuilder>>visitCommand: in Block: [ ...

```

Running tests render four errors:
![image](https://user-images.githubusercontent.com/1001427/165586029-8188a75e-77c5-42ae-a11d-c1aa8ba56d7a.png)

**Comment by dionisiydk** [comment]
Fixed in v6.0.4

**Comment by dionisiydk** [comment]
I fixed tests. But the issue with inspector integration on the level of Ghost.
GT was removed and it requires adaptation to new framework

**Comment by dionisiydk** [comment]
Fixed in v7.0.0
